src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -623,32 +631,76 @@ void FancyZones::ToggleEditor() noexcept
         return;
     }
 
+    winrt::com_ptr<IZoneWindow> zoneWindow;
+
     std::shared_lock readLock(m_lock);
-    auto zoneWindow = m_workAreaHandler.GetWorkArea(m_currentDesktopId, monitor);
+    
+    if (m_settings->GetSettings()->spanZonesAcrossMonitors)
+    {
+        zoneWindow = m_workAreaHandler.GetWorkArea(m_currentDesktopId, NULL);

```

Comment:
What does `GetWorkArea` return for `NULL` passed for monitor ?

Reply:
It returns the work area which spans all monitors.

Reply:
OK, there is an entry in the map with NULL as key for monitor? Since I haven't seen any changes in `MonitorWorkAreaHandler.cpp` and got confused.

Reply:
Yes, see `FancyZones.cpp:1008`